### ISSUE TITLE
diffs for mentors

### ISSUE THREAD

**Issue by kytrinyx** [exercism/exercism#3900]
_Imported from https://github.com/exercism/-DEPRECATED-PLEASE-USE-exercism.io-REPO-v2-feedback/issues/196._
_Originally posted by @NobbZ._

#

As a mentor, I'd like to see a diff between the curent and a previous iteration. probably even along with the comments given on the older submission.

**Comment by Stargator** [comment]
I heartedly agree with this request. When students post new iterations it would be very helpful to see from a glance what the changes were.

**Comment by mawis** [comment]
Me too. Some students don't really write anything else than code. It is hard to spot the changes they did. Escpecially if I hinted them to change some things, but to me the code looks the same again at first. A diff would help to see what they tried to change. With that I could better understand how they reacted to my hints.

**Comment by Emerentius** [comment]
I have manually committed a student's submissions into git to get this on an exercise where the submitted code was 400, later 200 lines.
I'm not sure how to communicate this to every student in an efficient way, but it would be nice if larger refactorings could be submitted piece-meal, one self-contained change after another.

**Comment by iHiD** [comment]
Which exercise required 400 LOC to solve?

**Comment by Emerentius** [comment]
I don't think it *requires* 400 LOC, but the Poker exercise invites some longer solutions, especially if people define their own sum types and implement traits for them (in Rust). It's a lot of boilerplate.

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂

**Comment by bobahop** [comment]
@kytrinyx , please reopen this issue

It would be nice to be able to see the diff between iterations for a solution submitted for mentoring. It's frequent enough that a student just submits another iteration without comment. It would be efficient if I could quickly see the difference from the iteration I last commented on, so I could focus feedback on what changed, instead of sifting through the whole thing and toggling back and forth looking for changes.
